docs/content/en/install/local/multi-cluster-helm/_index.md

```diff
@@ -263,9 +263,9 @@ kubectl -n k8ssandra-operator get deployment k8ssandra-operator \
 false
 ```
 
-### Generate and install ClientConfigs
+### Register the data plane to the control plane
 
-[create-clientconfig.sh](https://github.com/k8ssandra/k8ssandra-operator/blob/main/scripts/create-clientconfig.sh) lives in the k8ssandra-operator repo. It is used to configure access to remote clusters. 
+[k8ssandra-client](https://github.com/k8ssandra/k8ssandra-client) is used to register data planes to control planes. It does this by installing a ClientConfig on the control plane.
```

Comment:
**Suggestion:**
```suggestion
[k8ssandra-client](https://github.com/k8ssandra/k8ssandra-client) is used to register data planes to control planes. It does this by creating a ClientConfig custom resource on the control plane and a secret with a kubeconfig entry providing access to the dataplane.
```

Reply:
More detail than I think is strictly needed but it doesn't do any harm. Accepted.